_guides/how-we-do-development.md

```diff
@@ -127,7 +128,35 @@ opportunity to review, tidy up, and rearrange your commit history as needed).
 Rebasing can take some practice to get used to if you haven't done it often. Ask
 for help if you need it.
 
-### 7. Submit the changeset for code review
+### 7. Tidy up your commits
+
+Before requesting a review, consider tidying up your commit history on the
+branch to tell a clear, coherent story. We tend to review PRs commit by commit,
+as it allows you to see how the feature builds up on the branch, and having
+atomic, easily-digestible commits that logically flow from one to the next makes
+this a lot easier to review. Try to avoid touching unrelated code in the same PR
+- if you spot something that might need refactoring, do it in its own PR.
+
+Here’s an example of the sort of history you might want to tidy up:
+
+`bc870303 Add dashboard for viewing user profiles`
+
+`fd3ab401 Allow users to change passwords`
+
+`50a6a7a6 Fix typo on dashboard (can be squashed into bc870303)`
+
+`8b1cf775 Refactor password changing function (can be squashed into fd3ab401)`
+
+`f3344703 Fix flaky test introduced in previous PR (should probably be its own
+PR)`
```

Comment:
I suggest putting these all in a single fenced code block instead of multiple paragraphs.

Reply:
Yeah - I wanted to do this but was copying across from a google doc for some reason (didn't know how to use Netlify at first). Good suggestion!